sdk/attestation/test-resources-pre.ps1

```diff
@@ -6,6 +6,8 @@
 #Requires -Version 6.0
 #Requires -PSEdition Core
 
+# cspell: ignore asia australia australiasoutheast europe india iowa virginia
```

Comment:
I am surprised cspell is flagging these. Is it because of case sensitivity, it is thinking these should be capitalized as proper nouns?

Reply:
`australiaeast` is already in the global cspell.json list. Maybe add all these as well to it?

Reply:
I don't think we should capitalize them in `test-resources.ps1`, as these are the parameters for the powershell script, if I understand correctly, I think it is by design, and I don't have enough context to capitalize them here - it is likely that ARM would error.

As of `australiaeast`, I have 3rd PR in mind, to fix #4668. That would push some of the exclusions from `cspell.json` towards the files using them, if they are not widely used, and that PR would take care of `australiaeast` (unless it is used in 10s of files).